kolibri/plugins/device/assets/src/views/ManageContentPage/FilteredChannelListContainer.vue

```diff
@@ -73,14 +73,15 @@
     props: {
       channels: {
         type: Array,
+        default: () => [],
       },
       selectAllCheckbox: {
         type: Boolean,
         default: false,
       },
       selectedChannels: {
         type: Array,
-        required: false,
+        default: () => null,
```

Comment:
```suggestion
        default: () => [],
```

This also needs to be an array